styleguide/derek/layouts/leadCornerPocket.ejs

```diff
@@ -0,0 +1,16 @@
+<div class="container half-padding-full">
+  <div class="subpanel">
+    <div class="subpanel-inner">
+      <div class="row">
+        <div class="col-sm-9">
+          <%- partial('../_partials/solutions/lead.ejs') %>
+        </div>
+        <div class="col-sm-3">
+          <%- partial('../_partials/cards/corner-pocket.ejs', {
+            type: 'video'
+          }) %>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
Close, but the subpanels need to be inside the individual columns, not outside of it. 

```
<div class="container half-padding-full">
  <div class="row">
    <div class="col-sm-9">
      <div class="subpanel">
        <div class="subpanel-inner">
          <%- partial('../_partials/solutions/lead.ejs') %>
        </div>
      </div>
    </div>
    <div class="col-sm-3">
      <div class="subpanel">
        <div class="subpanel-inner">
          <%- partial('../_partials/cards/corner-pocket.ejs', {
            type: 'video'
          }) %>
        </div>
      </div>
    </div>
  </div>
</div>
```